source/CoreStartData.cpp

```diff
@@ -47,7 +47,7 @@ void CoreStartData::Save(DataWriter &out) const
 	out.BeginChild();
 	{
 		out.Write("system", system->Name());
-		out.Write("planet", planet->TrueName());
+		out.Write("planet", planet->Name());
```

Comment:
TrueName() was defined as the name to be used in the datafiles, why change to Name() here?

Reply:
Cause it was unnecessary. A planet's name is the name used in the datafiles. I could change it back, it doesn't really matter since `TrueName` just calls `Name`.